x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_custom_assets_extension.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { useKibana } from '../../../../../common/lib/kibana';
+import { APP_PATH } from '../../../../../../common/constants';
+import {
+  CustomAssetsAccordionProps,
+  CustomAssetsAccordion,
+  PackageAssetsComponent,
+} from '../../../../../../../fleet/public';
+
+export const EndpointCustomAssetsExtension: PackageAssetsComponent = () => {
+  const { http } = useKibana().services;
+  const views: CustomAssetsAccordionProps['views'] = [
+    {
+      name: 'Hosts',
+      url: http.basePath.prepend(`${APP_PATH}/hosts/allHosts`),
+      description: 'View hosts in Security app',
```

Comment:
This should probably say "View endpoints in Security app".

@caitlinbetz fyi

Reply:
also just thinking of this but it would make sense to have these strings translated

Reply:
Added translations in https://github.com/elastic/kibana/pull/103554/commits/86227cea8a27143288511b2da49980ca6cc276d5